src/e2e-test/features/gcs/sink/GCSSink.feature

```diff
@@ -265,3 +265,175 @@ Feature: GCS sink - Verification of GCS Sink plugin
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
     Then Verify data is transferred to target GCS bucket
+
+#Added new scenarios for GCS Sink - Bijay
+  @BQ_SOURCE_TEST @GCS_SINK_TEST
+  Scenario:Validate successful records transfer from BigQuery to GCS with macro enabled at sink
```

Comment:
Add the macro scenario in a separate feature file with name macro, refer other plugins feature file for naming convention.